src/components/Login.js

```diff
@@ -1,101 +1,242 @@
-import React, { useState } from "react";
-import Loginimage from "../images/Group 5.svg";
-import Logo from "../images/Logo.svg";
-import { Link, Redirect, Route, useHistory } from "react-router-dom";
+import React, { useState } from 'react'
+import Loginimage from '../images/Group 5.svg'
+import Logo from '../images/Logo.svg'
+import { useHistory, useLocation } from 'react-router-dom'
 
-const initialState = {
-  email: "",
-  password: ""
-};
+function Login(props) {
+  const history = useHistory()
+  if (localStorage.getItem('token')) {
+    history.push('/')
+  }
 
-function Login() {
-  const history = useHistory();
-  if (localStorage.getItem("token")) {history.push("/")}
-  
-  const [state, setState] = useState(initialState);
-  const handleChange = e => {
+  const resetPasswordMode = props.resetPasswordStatus
+  const keyValue = useLocation().search
+  const tokenParams = new URLSearchParams(keyValue)
+  const resetPasswordToken = tokenParams.get('token')
```

Comment:
Hmm you are doing some work here to get a `resetPasswordToken` but I can't see why you need it?
There is no query param in the URL to start with, which means `resetPasswordToken` is always _null_, unless I'm missing something.